rpmbuild/copr-rpmbuild.spec

```diff
@@ -103,32 +96,27 @@ Requires: dnf-yum
 Requires: dnf-utils
 %endif
 # selinux toolset to allow running ansible against the builder
-%if 0%{?fedora}
 Requires: python3-libselinux
 Requires: python3-libsemanage
-%else
-Requires: libselinux-python
-Requires: libsemanage-python
-%endif
+%if 0%{?openEuler}
 # for mock to allow: config_opts['nosync'] = True
 Requires: nosync
-%ifarch x86_64
-# multilib counterpart to avoid: config_opts['nosync_force'] = True
-Requires: nosync(x86-32)
 %endif
 Requires: openssh-clients
 Requires: podman
-Requires: pyp2rpm
-Requires: pyp2spec
+%if !0%{?openEuler}
+Suggests: pyp2rpm
+Suggests: pyp2spec
+Suggests: rubygem-gem2rpm
+Suggests: scl-utils-build
+Suggests: fedora-review >= 0.8
+Suggests: fedora-review-plugin-java
```

Comment:
Heh :) this is a nice misunderstanding.  We really need `Requires` here - but some suggestions on how to solve this kind of deployment-specific dependency requirements in general.